jerry-core/ecma/operations/ecma-typedarray-object.c

```diff
@@ -943,7 +943,8 @@ ecma_op_create_typedarray (const ecma_value_t *arguments_list_p, /**< the arg li
 
       ECMA_OP_TO_NUMBER_TRY_CATCH (num2, arg2, ret);
```

Comment:
The  `ECMA_OP_TO_NUMBER_TRY_CATCH` is unnecessary when using toLength operation.

Reply:
I made an error there the language specification says to use ToInteger there and not ToLength. So I've deleted my modifications there. On the other hand, I should have used ToLength somewhere else which I have added.